tags/tag_set_codec.go

```diff
@@ -188,7 +188,7 @@ func DecodeFromFullSignature(bytes []byte) (*TagSet, error) {
 
 	version := eg.readByte()
 	if version > tagsVersionID {
-		return nil, fmt.Errorf("DecodeFromFullSignature doesn't support version %v. Supports only up to: %v", version, tagsVersionID)
+		return nil, fmt.Errorf("not supported version: %q; supports only up to: %q", version, tagsVersionID)
```

Comment:
s/not supported/unsupported

how about 'unsupported tag version'

Reply:
Done

Yea, makes so much more sense :)